solidity/contracts/TokenStaking.sol

```diff
@@ -42,17 +42,16 @@ contract TokenStaking is Authorizations, StakeDelegatable {
     using Locks for Locks.Storage;
     using TopUps for TopUps.Storage;
 
-    event Staked(
-        address owner,
+    event StakeDelegated(
+        address indexed owner,
+        address indexed operator
+    );
+    event OperatorStaked(
```

Comment:
Not sure about the verbiage but couldn't find anything better than `StakeDelegated` + `OperatorStaked`.

Reply:
makes sense to me 👍